src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java

```diff
@@ -330,6 +330,21 @@ private void setAfterSubmitCallbackResponseEntity(final AfterSubmitCallbackRespo
         return metadata;
     }
 
+    @JsonIgnore
+    public Map<String, Object> getFormattedMetadata() {
```

Comment:
probably not the best name. You are not formatting here. You are changing the type of one of the map entries

Reply:
What would be recommended name ?

Reply:
I ended up using getConvertedMetadata